src/apiCalls.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+const fetchAllMovies = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+  .then(res => res.json())
```

Comment:
We put the third .then in the App because it has to do with those functions and we thought they should be together